iati/core/tests/test_default.py

```diff
@@ -240,3 +240,17 @@ def test_default_x_schema_modification_populated(self, default_call, codelist_no
 
         assert len(default_schema.codelists) == base_codelist_count + 1
         assert len(unmodified_schema.codelists) == base_codelist_count
+
+
+class TestDefaultVersions(object):
+    """A container for tests relating to default versions."""
+
+    def test_get_versions_by_integer(self):
+        result = iati.core.default.get_versions_by_integer()
+        all_versions = list(itertools.chain.from_iterable(result.values()))
+
+        assert sorted(result.keys()) == sorted(iati.core.constants.STANDARD_VERSIONS_MAJOR)
+        assert sorted(all_versions) == sorted(iati.core.constants.STANDARD_VERSIONS)
+        for integer_version, versions_in_integer in result.items():
+            for version in versions_in_integer:
+                assert version.startswith(str(integer_version))
```

Comment:
String operations are being done here... definitely makes more sense than the flooring and casting to define `STANDARD_VERSIONS_MAJOR`.